nipype/utils/filemanip.py

```diff
@@ -29,6 +29,12 @@
 fmlogger = logging.getLogger("filemanip")
 
 
+related_filetype_sets = [
```

Comment:
I take your point, but I think that leaving the periods makes the intent more obvious.

Would you suggest something like:

    related_filetype_sets = [tuple('.' + type_ext)
                             for type_ext in (
                                 ('hdr', 'img', 'mat'),
                                 ('BRIK', 'HEAD'),
                             )]

Or just handling it later (i.e., in a function `get_related_filetypes` that does what you suggest below)

I am still personally opting for leaving the dots in.